zerver/test_hooks.py

```diff
@@ -864,3 +864,24 @@ def test_travis_message(self):
                                        u"Details: [changes](https://github.com/hl7-fhir/fhir-sv"
                                        u"n/compare/6dccb98bcfd9...6c457d366a31), [build log](ht"
                                        u"tps://travis-ci.org/hl7-fhir/fhir-svn/builds/92495257)"))
+
+class YoHookTests(AuthedTestCase):
+    def test_yo_message(self):
+        """
+        Yo App sends notification whenever user receives a new Yo from another user.
+        """
+        email = "hamlet@zulip.com"
+        api_key = self.get_api_key(email)
+        body = ""
+
+        email = "hamlet@zulip.com"
+        sender = "IAGO"
+        ip = "127.0.0.1"
+        url = "/api/v1/external/yo?email=%s&api_key=%s&username=%s&user_ip=%s" % (email, api_key, sender, ip)
```

Comment:
Along with the url that we, as a user, add to Yo App's callback, Yo App sends two parameters : username and user_ip. So, this is the exact payload that we will be on the GET request.